services-directions/src/test/java/com/mapbox/api/directions/v5/models/LegStepTest.java

```diff
@@ -42,4 +43,52 @@ public void testSerializable() throws Exception {
     byte[] serialized = TestUtils.serialize(legStep);
     assertEquals(legStep, deserialize(serialized, LegStep.class));
   }
+
+  public void testToFromJson3() {
+
+    List<VoiceInstructions> voiceInstructions = Arrays.asList();
+    List<BannerInstructions> bannerInstructions = Arrays.asList();
+
+    List<StepIntersection> intersections =
+      Arrays.asList(StepIntersection.builder()
+      .in(0)
+      .out(1)
+      .entry(Arrays.asList(false, true, true))
+      .bearings(Arrays.asList(120, 210, 300))
+      .rawLocation(new double[]{13.424671, 52.508812})
+      .build());
+
+    legStep = LegStep.builder()
+      .distance(236.9)
+      .duration(59.1)
+      .geometry("asn_Ie_}pAdKxG")
+      .mode("driving")
+      .drivingSide("right")
+      .name("Adalbertstraße")
+      .weight(59.1)
+      .maneuver(StepManeuver.builder()
+        .rawLocation(new double[] {13.424671, 52.508812})
+        .bearingAfter(202.0)
+        .bearingBefore(299.0)
+        .type("turn")
+        .modifier("left")
+        .instruction("Turn left onto Adalbertstraße")
+        .build())
+      .intersections(intersections)
+      // TODO: Include the following
```

Comment:
Does this `TODO` need to be addressed before merging?

Reply:
@danesfeder  Thanks for flagging the TODOs.
I am not sure what kind of request would have those fields filled. Could you help me with those.